content/webapp/services/prismic/transformers/seasons.ts

```diff
@@ -1,14 +1,13 @@
-import { transformGenericFields } from '.';
+import { transformGenericFields, transformTimestamp } from '.';
 import { Season } from '../../../types/seasons';
 import { SeasonPrismicDocument } from '../types/seasons';
-import { parseTimestamp } from '@weco/common/services/prismic/parsers';
 
 export function transformSeason(document: SeasonPrismicDocument): Season {
   const { data } = document;
   const genericFields = transformGenericFields(document);
   const promo = genericFields.promo;
-  const start = parseTimestamp(data.start);
-  const end = data.end && parseTimestamp(data.end);
+  const start = transformTimestamp(data.start)!;
```

Comment:
How do we know it's safe to use the non-null assertion if the function can indeed return `undefined`?